Dockerfile

```diff
@@ -78,6 +78,7 @@ RUN pip install pip==24.2 setuptools==72.1.0 wheel==0.44.0
 # Jenkins image has a clean environment to start from).
 WORKDIR /tmp/katgpucbf
 COPY requirements.txt .
+COPY requirements-dev.txt .
```

Comment:
If we don't need this in the base, let's rather move it to the jenkins image section, so that changes to requirements-dev.txt don't invalidate the caches of other images.

Reply:
Fair point